src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -234,6 +236,10 @@ def remove_dialog_tag(text)
     link_to(text, '#', title: 'Close', id: 'remove_dialog')
   end
 
+  def close_dialog_tag(text, target)
+    link_to(text, '#', title: 'Close', class: 'close_dialog', data: { target: target })
```

Comment:
I created a helper, cause there was also one for `remove_dialog_tag`. I thought, it's more consistent.